dask_ml/model_selection/_incremental.py

```diff
@@ -56,17 +56,24 @@ def _score(model_and_meta, X, y, scorer):
         score = model.score(X, y)
 
     meta = dict(meta)
-    meta.update(score=score, score_time=time() - start)
+    meta["score_times"] += [time() - start]
```

Comment:
It looks like we keep a record of all previous times for each score.  I think that this isn't necessary.  I think that we keep all metas in a long running `history` variable on the client side that we should be able to use to reconstruct this later.  I think that this is probably slightly better because:

1.  We isolate the metadata to just a particular score
2.  We avoid passing around lists every time, which for longer runs might contribute to overhead?  (though not necessarily, I'm not sure how long these are likely to be)

Anyway, I don't think that this is necessary.  I would just rely on the history variable client-side.